### ISSUE TITLE
snp_match() - large % of alleles reversed

### ISSUE THREAD

**Issue by fsakhtari** [privefl/bigsnpr#135]
Hello,
When doing snp_match() between aT2D summary statistics and the 1000 Genome data, I see that almost 50% of SNPs are reversed.  This seems like a very large number and I'm wondering if this is to be expected or if I'm doing something incorrectly. Since the allele reversal determines the sign of the betas, this will affect the PRS and hence I want to ensure that I'm doing this correctly.
I'm using the datasets:
(A) Summary statistics - from this meta-analysis paper on T2D https://diabetes.diabetesjournals.org/content/diabetes/66/11/2888.full.pdf , downloaded from here: http://diagram-consortium.org/downloads.html - Scott et al., 2017
(B) LD reference panel - 1000 Genome Phase 1 dataset
Suggested QC was performed on the SNPs. I'm following the workflow described in the tutorial and in issue #125 

Following is the output:
"Match variants between (A) summary statistics and (B) 1000 Genomes Phase 1 LDrefpanel = (AB)"
167,419 variants to be matched.
9,784 ambiguous SNPs have been removed.
157,635 variants have been matched; 44 were flipped and 77,682 were reversed.   <<-- 50% variants reversed

I repeated this using the 1000 Genomes Phase 3 data as well and saw similar results
Following is the output :
"Match variants between (A) summary statistics and (B) 1000 Genomes Phase 3 LDrefpanel = (AB)"
150,623 variants to be matched.
8,882 ambiguous SNPs have been removed.
141,741 variants have been matched; 39 were flipped and 69,475 were reversed.   <<-- 50% variants reversed

I see that you've used this summary statistics data - Scott et al., 2017, in the LDpred2 paper with the UK Biobank data as the LD reference panel. Do you see similar number of variants reversed - could you share the snp_match() output? Is it possible for you to do a snp_match with the Scott et al. summary statistics and the 1000 Genome data and share the output?

Thank you,
Farida

**Comment by privefl** [comment]
Yes, seems I get something similar:
https://github.com/privefl/paper-ldpred2/blob/master/code/prepare-sumstats.R#L177

**Comment by fsakhtari** [comment]
Thanks for checking, Florian.
I used the UK biobank data as the LD reference panel and did the snp_match and see numbers same as yours:
"Match variants between (A) summary statistics and (B) UK Biobank  data as LDrefpanel = (AB)"
179,575 variants to be matched.
10,661 ambiguous SNPs have been removed.
168,914 variants have been matched; 46 were flipped and 82,090 were reversed.

In the link that you sent, I see that just as high % of alleles are reversed for all summary statistics except T1D, BRCA & Asthma. So, this is ok then, I guess?
